src/main/java/Milo.java

```diff
@@ -0,0 +1,135 @@
+import java.util.Scanner;
+
+public class Milo {
+    private static final int MAX_TASKS = 100;
+    private static Task[] tasks = new Task[MAX_TASKS];
+    private static int count = 0;
+
+    public static void main(String[] args) {
+        printLogo();
+        run();
+    }
+
+    private static void run() {
+        Scanner sc = new Scanner(System.in);
+        String command = sc.nextLine().trim().toLowerCase();
+        while (!command.equals("bye")) {
+            if (command.equals("list")) {
+                printList();
+            } else if (command.startsWith("mark")) {
+                markTask(command);
+            } else if (command.startsWith("unmark")) {
+                unmarkTask(command);
+            } else if (command.startsWith("todo")) {
+                todoTask(command);
+            } else if (command.startsWith("deadline")) {
+                deadlineTask(command);
+            } else if (command.startsWith("event")) {
+                eventTask(command);
+            } else {
+                addTask(command);
+            }
+            command = sc.nextLine();
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    private static void printLogo() {
+        String logo = " __  __ _ _       \n"
+                + "|  \\/  (_) | ___  \n"
+                + "| |\\/| | | |/ _ \\ \n"
+                + "| |  | | | |  __/ \n"
+                + "|_|  |_|_|_|\\___| \n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?\n");
+    }
+
+    private static void printList() {
+        System.out.println("  ____________________________________________________________");
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < count; i++) {
+            String status = tasks[i].isDone ? "X" : " ";
+            System.out.println("  " + (i + 1) + "." + tasks[i].toString());
+        }
+        System.out.println("  ____________________________________________________________");
+    }
+
+    private static void markTask(String command) {
+        String[] parts = command.split(" ");
+        int index = Integer.parseInt(parts[1]) - 1;
+        tasks[index].isDone = true;
+        System.out.println("  ____________________________________________________________");
```

Comment:
Could there be a line spacing between segments of the code so it is like a logical structure?

```suggestion
        tasks[index].isDone = true;
        
        System.out.println("  ____________________________________________________________");
```